content/docs/demos/prometheus_grafana.md

```diff
@@ -285,16 +285,17 @@ Here you should see most of the endpoints connected, up and running for scrape.
 
 ## Deploying a Grafana Instance
 
-Similar to Prometheus, we will deploy a Grafan instance through helm.
+Similar to Prometheus, we will deploy a Grafana instance through helm.
 
 ```
 helm repo add grafana https://grafana.github.io/helm-charts
-helm install stable/grafana --generate-name
+helm install grafana/grafana --generate-name
 ```
 
-Next grab Grafana's admin password
+Next grab Grafana's admin password:
+
 ```
-kubectl get secret --namespace <grafana namespace> <grafana pod> -o jsonpath="{.data.admin-password}" | base64 --decode ; echo
+kubectl get secret --namespace <grafana namespace> <grafana secret> -o jsonpath="{.data.admin-password}" | base64 --decode ; echo
```

Comment:
Good catch, could you clarify that the grafana secret name is the generated name from the previous `helm install` command? The output of `helm install grafana/grafana --generate-name` should also include instructions for getting the admin password, but it would be helpful clarify what the secret name should be and how to get it.

Reply:
HI @jaellio, I've modified it a bit to use the same schema that's used for the pod name:

```
export GRAFANA_NAMESPACE=<grafana namespace> # kubernetes namespace grafana was installed to (can be default)
export SECRET_NAME=$(kubectl get secret --namespace $GRAFANA_NAMESPACE -l "app.kubernetes.io/name=grafana" -o jsonpath="{.items[0].metadata.name}")
kubectl get secret --namespace $GRAFANA_NAMESPACE $SECRET_NAME -o jsonpath="{.data.admin-password}" | base64 --decode ; echo
```

WDYT?

Reply:
My only problem with that is we are getting the same secret data twice. Once to get the name, and a second time to get the admin password.

The following combines the commands:

```
export grafana_namespace=default # Replace default if Grafana was deployed to a different namespace. 
kubectl get secret --namespace $grafana_namespace -l "app.kubernetes.io/name=grafana" -o jsonpath="{.items[0].data.admin-password}" | base64 --decode ; echo
```

Alternatively, on `helm install` we could set the `namespaceOverride` parameter.

```
export grafana_namespace=default # Replace default with the namespace where Grafana should be deployed
helm install grafana/grafana --set namespaceOverride=$grafana_namespace --generate-name

kubectl get secret --namespace $grafana_namespace -l "app.kubernetes.io/name=grafana" -o jsonpath="{.items[0].data.admin-password}" | base64 --decode ; echo
```

WDYT?

Reply:
@heoelri, are still able to make these changes? We would love to include them in our latest release site.